server/db/seed.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * seed script to create user so they can login
+ */
+const mongoose = require("mongoose");
+
+require("../../models/User");
+
+const User = mongoose.model("User");
+
+mongoose.connect(
+	process.env.MONGODB_URI || "mongodb://localhost/WowRealtor",
+	async (error) => {
+		await User.create({ email: "admin@gmail.com", password: "123123" });
+		await User.create({ email: "alex@gmail.com", password: "123123" });
```

Comment:
seed script to add 2 email to the database
add your email in if you want for testing or use the UI to register

this script is run with:
`$ npm run seed`